app/res/layout/media_layout.xml

```diff
@@ -1,12 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
 <merge xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto">
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <!-- Used to place media views to the top of the layout. -->
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/top_guideline"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        app:layout_constraintGuide_begin="0dp"/>
 
     <org.commcare.views.media.AudioPlaybackButton
         android:id="@+id/audio_button"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:visibility="gone"
+        tools:visibility="gone"
```

Comment:
tools:visibility should be `visible` while keeping the `android:visibility` as `gone`

Reply:
Ahh Damn, silly me! I thought it behaves the same as `android:visibility`